python_modules/libraries/dagster-azure/dagster_azure/adls2/file_manager.py

```diff
@@ -106,10 +113,15 @@ def write(self, file_obj, mode="wb", ext=None):
             file_system=self._file_system, file_path=adls2_key
         )
         adls2_file.upload_data(file_obj, overwrite=True)
-        return ADLS2FileHandle(self._client.account_name, self._file_system, adls2_key)
+        return ADLS2FileHandle(
+            self._client.account_name, self._file_system, adls2_key, self._client.primary_endpoint
+        )
 
     def get_full_key(self, file_key):
         return f"{self._prefix}/{file_key}"
 
     def delete_local_temp(self):
         self._temp_file_manager.close()
+
+    def get_client(self):
+        return self._client
```

Comment:
This was only used in the test suite and since I've edited the test to access the private attribute directly, this is no longer needed and has been removed.